tasks/libs/junit_upload.py

```diff
@@ -24,7 +24,9 @@ def split_junitxml(xml_path, codeowners, output_dir):
     for suite in tree.iter("testsuite"):
         path = suite.attrib["name"].replace(REPO_NAME_PREFIX, "", 1)
 
-        owners = codeowners.of(path)
+        # Dirs in CODEOWNERS might end with "/", but testsuite names in JUnit XML
```

Comment:
Does this still work when directory paths in CODEOWNERS don't end in `/`?

Reply:
Yeah, it does. If you have `/foo/bar` in the CODEOWNERS file and you ask for ownership of `/foo/bar/`, it will still match.